package.json

```diff
@@ -36,7 +36,8 @@
     "build": "lerna run build",
     "prepare": "husky install",
     "deploy": "lerna publish from-package --no-private --no-verify-access --conventional-commits --conventional-prerelease=@mux-elements/mux-player,@mux-elements/mux-player-react",
-    "version:update": "lerna version --exact --no-private --conventional-commits --conventional-prerelease=@mux-elements/mux-player,@mux-elements/mux-player-react --create-release github"
+    "version:update": "lerna version --exact --no-private --conventional-commits --conventional-prerelease=@mux-elements/mux-player,@mux-elements/mux-player-react --create-release github",
```

Comment:
I think we don't want `--create-release github` anymore?

Reply:
yup, I thought first to try the release-notes GH action out separately. if it works good, I'll remove in next PR

Reply:
sounds good to me